File: FizzBuzz/Python3/fizzbuzz.py
def fizzbuzz(i):
    if i % 15 == 0:
        print("FizzBuzz!")
    elif i % 3 == 0:
        print("Fizz")
    elif i % 5 == 0:
        print("Buzz")
    else:
        print("")

[ fizzbuzz(i) for i in range(1, 100) ]

File: FizzBuzz/javascript/fizzbuzz.js
var APP = APP || {};

APP.arr = Array.apply(null, new Array(100)).map((v, i) => i+1);

APP.fizzbuzz = i => {
    if (i % 15 === 0) {
        return "FizzBuzz!";
    } else if (i % 3 === 0) {
        return "Fizz";
    } else if (i % 5 === 0) {
        return "Buzz";
    } else {
        return i.toString(10);
    }
}

APP.arr.map(APP.fizzbuzz).forEach(console.log);